Sources/ScipioKit/DescriptionPackage.swift

```diff
@@ -69,7 +65,8 @@ struct DescriptionPackage {
         let workspace = try Workspace(
             fileSystem: fileSystem,
             location: Workspace.Location(forRootPackage: packagePath, fileSystem: fileSystem),
-            configuration: workspaceConfiguration
+            configuration: workspaceConfiguration,
+            customHostToolchain: toolchain
```

Comment:
Is there some reason why `customHostToolchain` was added this time even though it is a optional customization?

Reply:
Because, SwiftPM internal workspace tried to generate another user toolchain.
It should be shared.